test_pad_op (Failed)

test_pad_op fails randomly in Teamcity, can adjust the max_relative_error?
```
[12:23:03]	149/161 Test #150: test_pad_op .........................***Failed 1.51 sec
[12:23:03]	F.......
[12:23:03]	======================================================================
[12:23:03]	FAIL: test_check_grad_normal (__main__.TestCase1)
[12:23:03]	----------------------------------------------------------------------
[12:23:03]	Traceback (most recent call last):
[12:23:03]	File "test_pad_op.py", line 25, in test_check_grad_normal
[12:23:03]	self.check_grad(['X'], 'Out')
[12:23:03]	File "/paddle/python/paddle/v2/framework/tests/op_test.py", line 268, in check_grad
[12:23:03]	"Gradient Check On %s" % str(cpu_place))
[12:23:03]	File "/paddle/python/paddle/v2/framework/tests/op_test.py", line 227, in __assert_is_close
[12:23:03]	self.assertLessEqual(max_diff, max_relative_error, err_msg())
[12:23:03]	AssertionError: Gradient Check On CPUPlace Variable X@GRAD max gradient diff 0.005153 over limit 0.005000, the first error element is 56
```

--- a/python/paddle/v2/framework/tests/test_pad_op.py
+++ b/python/paddle/v2/framework/tests/test_pad_op.py
@@ -22,7 +22,7 @@ class TestPadOp(OpTest):
         self.check_output()
 
     def test_check_grad_normal(self):
-        self.check_grad(['X'], 'Out', max_relative_error=0.5)
+        self.check_grad(['X'], 'Out', max_relative_error=0.006)
 
     def initTestCase(self):
         self.shape = (16, 16)

--- a/python/paddle/v2/framework/tests/test_pad_op.py
+++ b/python/paddle/v2/framework/tests/test_pad_op.py
@@ -22,7 +22,7 @@ class TestPadOp(OpTest):
         self.check_output()
 
     def test_check_grad_normal(self):
-        self.check_grad(['X'], 'Out')
+        self.check_grad(['X'], 'Out', max_relative_error=0.5)
 
     def initTestCase(self):
         self.shape = (16, 16)

